src/diffusers/models/unet_2d_condition_kandi3.py

```diff
@@ -1,16 +1,28 @@
-import math
```

Comment:
Actually can we rename this file to `unet_kandinsky3.py` ? I don't like `kandi..` much